iis-workflows/iis-workflows-ingest-pmc/src/main/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/MetadataImporter.java

```diff
@@ -46,15 +85,29 @@ protected void map(AvroKey<DocumentText> key, NullWritable value, Context contex
 				saxParser.parse(new InputSource(
 						new StringReader(nlm.getText().toString())), 
 						pmcXmlHandler);
-                context.write(new AvroKey<ExtractedDocumentMetadata>(output.build()), 
-                		NullWritable.get());
+				mos.write(namedOutputMeta, 
+						new AvroKey<ExtractedDocumentMetadata>(output.build()));
         	 } catch (ParserConfigurationException e) {
-             	throw new IOException("Text extraction failed for id " + nlm.getId() + 
-             			" and text: " + nlm.getText(), e);
+        		 Map<CharSequence, CharSequence> auditSupplementaryData = new HashMap<CharSequence, CharSequence>();
+        			auditSupplementaryData.put(FAULT_TEXT, nlm.getText());
```

Comment:
I could go with the list of supported keys but would it be meaningful without mentioning in which context they do appear? 

Mentioning about context will lead to comments related to other modules such as:
* `processing_time` - provided by pdf metadataextraction
* `url` - used by pdf metadataextraction to point to the source url of the content
* `text` -  used by pmc ingester, conveys pmc source xml file

what doesn't look good in such generic module as `iis-schemas`.

Besides, `auditSupplementaryData` is self explanatory - this is an auxiliary data that can be helpful in debugging fault, but it is not an essential part of `Fault` model. 

Maybe fault dedicated wiki page will be a better place for such details?